hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ECUnderReplicationHandler.java

```diff
@@ -51,7 +50,7 @@
  * Handles the EC Under replication processing and forming the respective SCM
  * commands.
  */
-public class ECUnderReplicationHandler implements UnderReplicationHandler {
+public class ECUnderReplicationHandler implements UnhealthyReplicationHandler {
```

Comment:
Not sure we will call over replication as unhealthy. Don't have better name in my mind but how about simply ReplicationHandler ?


Since it is common class now, javadoc should represent the generic message. Currently it assumed for underReplication handler.

Reply:
`ReplicationHandler` seems a little confusing, it might be considered as the handler to handle replication command.
so i suggest we use `unhealthy` just for now and change it to an appropriate name after we find a proper one.  what about "nonOptimalReplicationHandler"